CHANGELOG.md

```diff
@@ -8,8 +8,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ## [Unreleased]
 
 ### Added
+* Make metrics quantile collector age params configurable (#286).
 
 ### Changed
+* Display total average in `crud.stats` if quantile is `-nan` (#286).
```

Comment:
Cool, but may be it's better to add average as additional field of stats and leave latency as before? This will make possible to see overall average and also quantile in a period of time of 'max_age_time'

Reply:
For now there are three possible approaches.

### Fill `-nan` with average
(implemented now)

#### Pros
- API remains the same, backward compatible (in all adequate cases)
- The logic behind the values changes slightly

#### Cons
- No possible way to distinct what is displayed in `latency` now
- Whole `latency` logic is really complicated and hard to understand

### Add `average` field, do not change `latency`

#### Pros
- Backward compatible
- When quantiles enabled, possible to see both average and quantile latency

#### Cons
- The logic is still confusing since in many cases `average` = `latency` (local driver or quantiles disabled)


### Add `average` field, display only quantile value in `latency`

#### Pros
- Logic is rather clear and simple to understand
- When quantiles enabled, possible to see both average and quantile latency

#### Cons
- Breaking change

Options 2 and 3 really sounds more satisfying than 1. I think we'll discuss it a bit and choose a solution.

Reply:
One more possible approach

### Add separate `average` and `quantile` fields, deprecate (but not remove yet) `latency` field

It should clear up the logic behind the values and smooth up `latency` breaking change.

Reply:
It is implemented as suggested by Yaroslav, so I'll resolve the discussion.